src/main/java/macrobase/analysis/result/OutlierClassificationResult.java

```diff
@@ -0,0 +1,4 @@
+package macrobase.analysis.result;
+
+public abstract class OutlierClassificationResult {
```

Comment:
I'm in favor of interfaces, but they probably make the most sense if we have a clear set of well-defined functions like `getScore()` `getDatum()` `isOutlier()`, etc. An interface / abstract class without any members or member functions seems slightly gratuitious.

So, what do you think about having an interface `OutlierClassificationResult` that has the above methods?